notebooks/DatasetVisualization_pascalvoc.ipynb

```diff
@@ -0,0 +1,306 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "###################################################################################################\n",
+    "#\n",
+    "# Copyright (C) 2023 Maxim Integrated Products, Inc. All Rights Reserved.\n",
+    "#\n",
+    "# Maxim Integrated Products, Inc. Default Copyright Notice:\n",
+    "# https://www.maximintegrated.com/en/aboutus/legal/copyrights.html\n",
+    "#\n",
+    "###################################################################################################\n",
+    "\n",
+    "import os\n",
+    "import sys\n",
+    "\n",
+    "import numpy as np\n",
+    "import torch\n",
+    "\n",
+    "import matplotlib.patches as patches\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "sys.path.append(os.path.dirname(os.getcwd()))\n",
+    "sys.path.append(os.path.join(os.path.dirname(os.getcwd()), 'models'))\n",
+    "\n",
+    "from datasets import pascalvoc"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create test and train sets and visualize sample item/s"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Train dataset length: 16551\n",
+      "\n",
+      "Test dataset length: 4952\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "device = \"cpu\"\n",
+    "data_path = '/data_ssd/'\n",
```

Comment:
We should mark this as "CHANGE ME TO DATA DIRECTORY", remove at least the trailing slash and then use os.path.join later